src/components/WidgetWeekWeather.vue

```diff
@@ -0,0 +1,127 @@
+<script setup lang="ts">
+import { invoke } from "@tauri-apps/api/core";
+import { listen } from '@tauri-apps/api/event';
+import { computedAsync } from "@vueuse/core";
+import { ref, triggerRef, watch } from "vue";
+import { Settings } from "../types";
+import { resolveResource } from '@tauri-apps/api/path';
+import { readTextFile } from '@tauri-apps/plugin-fs';
+
+const resourcePath = await resolveResource('city_code.json');
+const cities = JSON.parse(await readTextFile(resourcePath));
```

Comment:
あとこいつにも型付けたいから
```ts
type Cities = { [key: string]: { name: string; lnglat: [number, number]; } };
```
みたいなのつくって(これも多分types.tsに書いたほうがいい)
```suggestion
const cities = JSON.parse(await readTextFile(resourcePath)) as Cities;
```
しよう